### DISCUSSION TITLE
关于claude2

### DISCUSSION THREAD

**Discussion by M3ngZ** [swuecho/chat#382] [Q&A]
看release上有更新claude2的支持，请问要怎么设置claude2呢，我看模型列表里似乎没有

**Comment by swuecho** [comment]
https://github.com/swuecho/chat/blob/master/api/sqlc/schema.sql#L43C8-L43C130


<img width="1785" alt="image" src="https://github.com/swuecho/chat/assets/666683/b91a28ae-eedd-4e85-a209-74183df88d6c">

有啊. Claude 就是claude2
